pkg/config/setup/apm.go

```diff
@@ -85,8 +85,29 @@ func setupAPM(config pkgconfigmodel.Setup) {
 	config.BindEnvAndSetDefault("apm_config.compute_stats_by_span_kind", true, "DD_APM_COMPUTE_STATS_BY_SPAN_KIND")                           //nolint:errcheck
 	config.BindEnvAndSetDefault("apm_config.instrumentation.enabled", false, "DD_APM_INSTRUMENTATION_ENABLED")
 	config.BindEnvAndSetDefault("apm_config.instrumentation.enabled_namespaces", []string{}, "DD_APM_INSTRUMENTATION_ENABLED_NAMESPACES")
+	config.ParseEnvAsStringSlice("apm_config.instrumentation.enabled_namespaces", func(in string) []string {
```

Comment:
https://github.com/DataDog/helm-charts/blob/fa73f62265f91c9d55685925f769a43afd870ba3/charts/datadog/templates/cluster-agent-deployment.yaml#L272-L279 This is all fine since we quote the json and pass the string through right?

Reply:
Looks correct to me. From the config's perspective, env vars always have string values, and since the value in the helm chart is converted to json, this handler will properly parse it to a slice of strings.